database/CorePartitionManager.java

```diff
@@ -127,26 +126,26 @@ ColumnFamilyHandle get(Key.Partition partition) {
                     configuration.defaultCFOptions()
             );
             descriptors[VARIABLE_START_EDGE_HANDLE_INDEX] = new ColumnFamilyDescriptor(
-                    ByteArray.encodeString(VARIABLE_START_EDGE.name(), STRING_ENCODING).getBytes(),
+                    VARIABLE_START_EDGE.encoding().partitionName().get().getBytes(),
```

Comment:
Declare column families with partition name defined in the `Encoding` class instead using the `name()` built-in function. Using the latter is unsafe - if we refactor the enum declaration, the value returned by `name()` will change accordingly.